pkg/binding/selectors.go

```diff
@@ -82,14 +89,14 @@ func (c *Controller) updateResolutions(ctx context.Context, obj runtime.Object)
 
 		if resolutionUpdated {
 			// enqueue binding to be synced since an object was added to its bindingpolicy's resolution
-			c.logger.V(4).Info("enqueued Binding for syncing due to a noting of an "+
+			logger.V(4).Info("enqueued Binding for syncing due to a noting of an "+
```

Comment:
Missed this one, should be capitalized too.

Reply:
Done.